.travis.yml

```diff
@@ -1,19 +0,0 @@
- 
-language: python
-
-sudo: False
-dist: trusty
-
-python:
-    - 2.7
-    - 3.6
-
-install:
-    - pip install -U pip
-    - pip install -r requirements.txt
-    - pip install coveralls
-    - pip install -e .
-
-script: skip
-
-after_success: coveralls
```

Comment:
why delete this?

Reply:
Oh, I thought we didn't need it anymore? We aren't using travis CI. I can put it back if it is needed.

Reply:
fair, we're invested in actions now.